### DISCUSSION TITLE
public 객체들이 실행만하면 리셋됨

### DISCUSSION THREAD

**Discussion by shogun0331** [naverz/zepeto-studio-kor#166] [Issue Report (Legacy)]
![error](https://user-images.githubusercontent.com/25839676/148172929-d0d1431a-1230-4110-9594-f1b367d8b288.png)

가이드대로 따라하고 실행하면  모두 날라갑니다 저만 그런가요?
혹시나 해서 씬저장도 해보고 껏다도 켜보고 다했습니다.

**Comment by x4cf** [comment]
현재 배열을 선언하고 하나라도 None인 상태로 저장을하면 고장납니다. 배열을 사용할만큼 선언하고 None이 없도록 유의해주어야 합니다

**Comment by moondory77-naverz** [comment]
안녕하세요.
현재 보여주신 스크린샷 만으로는, 문제의 원인을 파악하기가 어려운듯 합니다.
조금만 더 문제상황의 단서를 더 주시면, 구체적인 도움을 드릴 수 있도록 하겠습니다.

해당 상황이 벌어 질 때의 에러로그
사용 중인 ZEPETO.World 버전
추가로, 문제상황(스크립트 프로퍼티가 Play 시, 모두 초기화 되는 현상) 을 재현할 수 있는 방법

감사합니다.

**Reply by moondory77-naverz** [reply]
@shogun0331 님.
현재 문제 상황에서, 사용 중인 ZEPETO.World 패키지 버전을 알려주시면, 빠르게 문제상황을 확인하고 도움드릴 수 있을 듯 합니다.
감사합니다.
